apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -247,24 +261,23 @@ export type FetchResearchOutputOptions = FetchOptions<ResearchOutputFilter> & {
   includeDrafts?: boolean;
 };
 
-const makeODataFilter = (filter?: ResearchOutputFilter): string => {
+const makeODataFilter = (filter?: ResearchOutputFilter): Filter | null => {
```

Comment:
although im generally against that i think it would be helpful to export this function and write some unit tests for it. It's just a little hard to read right now so I'm interested to see what the input and output should look like